src/meshapi/tests/test_join_form.py

```diff
@@ -961,7 +961,7 @@ def test_member_moved_and_used_non_primary_email_join_form(self):
             f"but got {response3.status_code}.\n Response is: {response3.content.decode('utf-8')}",
         )
 
-        validate_successful_join_form_submission(self, "Valid Join Form", s, response3, expected_member_count=2)
+        validate_successful_join_form_submission(self, "Valid Join Form", s, response3, expected_member_count=3)
```

Comment:
It's 5AM so I'm making a wild fucking guess, but this should be making 3 instead of 2 now, since phone number is no longer deduped